### ISSUE TITLE
Add statistics page

### ISSUE THREAD

**Issue by tlammer** [specify/specify7#501]
SDSU requested having a statistics page like there is in 6 to track numbers.

**Comment by maxxxxxdlp** [comment]
The current workaround would be for them to create some sort of HTML page that would get data from Specify APIs and display statistics.

Then, they can replace the welcome page with their custom statistics page
Instructions on how to customize the welcome page: https://github.com/specify/specify7/wiki/Customizing-the-splash-screen.

**Comment by tlammer** [comment]
also requested by Cleveland

**Comment by maxxxxxdlp** [comment]
At the very bottom of the Specify Network page, there is a link to see collection statistics over time

**Comment by maxxxxxdlp** [comment]
A simple collection statistics page has been introduced in Specify 7.6

To access it:
1. Open any Collection Object record that is uploaded to GBIF or iDIgBio
2. Click on "Specify Network"
3. Scroll to the bottom of the page
4. Click on "Collection Statistics"

**Comment by benanhalt** [comment]
@realVinayak is working on supporting the existing Specify 6 stats page in 7.

**Comment by grantfitzsimmons** [comment]
- Add percentage of records georeferenced
- Attachment counts

**Comment by realVinayak** [comment]
Nooo. Let's reopen till release :)

**Comment by realVinayak** [comment]
also don't see this fixed in production right now. We need to be very serious when closing issues @grantfitzsimmons, and make sure they can't be replicated on production. /s

**Comment by grantfitzsimmons** [comment]
Our workflow has been that we close issues when they are complete on a milestone (see https://github.com/specify/specify7/milestone/51). Do you think that's inadvisable?

**Comment by maxxxxxdlp** [comment]
yes, I would suggest putting issues into release milestone and closing them as soon as they are tested
benefit:
1. fewer open issues - less noise
2. easy to see how much progress has been done and how much is remaining (see the progress bars here: https://github.com/specify/specify7/milestones)
